sdk/python/setup.py

```diff
@@ -46,6 +46,8 @@
     "numpy",
     "google",
     "confluent_kafka",
+    "grpcio-opentracing",
+    "opentracing"
```

Comment:
Do we need to force users to have this dependency?

Reply:
This is a little tricky. `opentracing` is only used because i want to use type hint for the tracer argument. So, if i remove the type hint, we don't really need `opentracing`, and the dependency can be safely removed.

As for `grpcio-opentracing`, that is used for the interceptor. I can move the import within the method (and only if tracer is set), so that the SDK doesn't run into runtime error. But, how should the user know that they need to include `grpcio-opentracing` in their dependencies, if they want to enable tracing? Would returning an error message and document this explicitly in the method comment suffice?